docs/reference/overview.md

```diff
@@ -31,6 +38,12 @@ Setting this is optional. If you do not set this, the `COMPOSE_PROJECT_NAME` def
 
 Specify the file containing the compose configuration. If not provided, Compose looks for a file named  `docker-compose.yml` in the current directory and then each parent directory in succession until a file by that name is found.
 
+### COMPOSE\_API\_VERSION
+
+Specify the api version the client should report. If you get an error about
+`client and server don't have same version` you may use this environment
+variable to have the client report a version compatible with the server.
```

Comment:
"matching" sounds good.

>  Which features do I lose if I do do it.

That will depend on the version of the client and the server. It will be different for basically every combination, so we can't accurately enumerate them here.

> Why wouldn't I just fix the problem with the client/API mismatch

Sometimes (especially at larger orgs, or when there are shared hosts where the users don't necessarily have root) it takes a lot longer to upgrade the server. The server might only be upgraded every second or third version, where as a developer can easily upgrade the client (docker-compose) at any time.

So this environment variable is a workaround for when you want bug fixes or new features from the client, but you can't get a newer server just yet.